db/schema.rb

```diff
@@ -509,6 +509,7 @@
   add_index "listing_units", ["listing_shape_id"], name: "index_listing_units_on_listing_shape_id", using: :btree
 
   create_table "listings", force: :cascade do |t|
+    t.binary   "uuid",                            limit: 16
```

Comment:
Here is the relevant MySQL adapter code that converts the Rails types to MySQL types:

https://github.com/rails/rails/blob/4b5570bd1bdd6c6750ae06ace1806f9cafd68f12/activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb#L573

@ovan  Should we just use the `VARBINARY` type generated by the `:binary, limit: 16` Rails type?